src/core/thread/mesh_forwarder.cpp

```diff
@@ -2271,4 +2279,75 @@ void MeshForwarder::HandleDataPollTimeout(void)
     }
 }
 
+#if OPENTHREAD_CONFIG_LOG_LEVEL >= OPENTHREAD_LOG_LEVEL_INFO
```

Comment:
It's just that on Windows, lots of traffic can get generated, and I'm not sure we want to log every packet if just INFO level is enabled. I think it is fine for now. If it really becomes an issue later, we can work something out.